src/Modules/ElementUtils/inheritance.js

```diff
@@ -6,13 +6,13 @@ const INHERITED_MEMBER_QUERY = {
   $or: [
     {
       'meta.links': {
-        href: 'http://refract.link/inherited-member/',
+        href: 'http://example.com/inherited-member/',
```

Comment:
I don't think it has any use as of now and that domain was registered because of this mention in the codebase.